src/main/java/uk/gov/hmcts/divorce/notification/FormatUtil.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.divorce.notification;
+
+import java.time.format.DateTimeFormatter;
+
+import static java.lang.String.join;
+import static java.time.format.DateTimeFormatter.ofPattern;
+
+public final class FormatUtil {
+
+    public static final DateTimeFormatter dateTimeFormatter = ofPattern("d MMMM yyyy");
+
+    private FormatUtil() {
+    }
+
+    public static String formatId(final Long id) {
+        return join("-", id.toString().split("(?<=\\G....)"));
```

Comment:
I don't think we need to format(remove hyphens) in the callback we receive just a long number as a string I think

Reply:
This method adds a hyphen after each 4 numbers, Linus added it for setting the reference in the citizen submit email so I maded it available as a common util. Output is - 1234-5678-9012-3456

Reply:
ohh got it